src/components/Admin/Toolbar.jsx

```diff
@@ -72,9 +92,35 @@ const Toolbar = ({ input, setInput, tags, setObjects, objects, filters }) => {
     );
     setToggle(!toggle);
   };
+  const blacklist = ["uid", "selected", "hidden", "links", "dropdown"];
+  const mapObjectsToCSVData = (objects, blacklist, headers) => {
+    const data = [];
+    const columnNames = headers
+      .map((header) => header.text)
+      .filter((key) => key && !blacklist.includes(key));
+    data.push(columnNames);
+    objects.forEach((item) => {
+      const rowData = {};
+      for (const key of columnNames) {
+        const value = item[key];
+        rowData[key] = convert(value);
+      }
+      data.push(Object.values(rowData));
```

Comment:
if u just need an array, then just store it as an array originally, it should remove the need for Object.values